src/test_l1_dsAudio.c

```diff
@@ -161,8 +161,11 @@ void test_l1_dsAudio_negative_dsAudioPortInit (void)
 
 	// Step 02: Call dsAudioPortInit() Attempt to initialize dsAudio again
 	result = dsAudioPortInit();
+#ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.

Reply:
Done